lib/app.coffee

```diff
@@ -14,6 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ###
 
+# Assign bluebird as the global promise library
+# stream-to-promise will produce native promises if not
+# for this module, which could wreak havoc in this
+# bluebird-only codebase.
+require('any-promise/register/bluebird')
```

Comment:
This statement relocation (from line 86 to line 21) solved an error during testing:
```
Error: any-promise already defined as "global.Promise".  You can only register an implementation before the first  call to require("any-promise") and an implementation cannot be changed
```
I didn't really understand why it necessary for the v8 branch, and not for the v9 / v10 / v11 branches. I assume it's to do with dependency versions (older CLI versions did not have a npm-shrinkwrap file). But the change seems harmless, and passed some basic tests.